CHANGELOG.md

```diff
@@ -118,6 +118,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Adds /version endpoint to distribution API
 - **CUMULUS-2497**
   - Created `isISOFile()` to check if a CMR file is a CMR ISO file.
+- **CUMULUS-2301**
```

Comment:
These CHANGELOG notes need to be moved. They're currently nested under release 9.1.0 notes but should be under "Unreleased"

Reply:
[`2c9bd18` (#2294)](https://github.com/nasa/cumulus/pull/2294/commits/2c9bd186888194930101931e267116772c417d7c)